cl/search/models.py

```diff
@@ -1896,6 +1902,72 @@ class OpinionCluster(models.Model):
         "outcome.",
         blank=True,
     )
+    headnotes = models.TextField(
+        help_text="Headnotes are summary descriptions of the legal "
+        "issues discussed by the court in the particular case. "
+        "They appear at the beginning of each case just after "
+        "the summary and disposition. "
+        "They are short paragraphs with a heading in bold face type."
+        " From Wikipedia - A headnote is a brief summary of a "
+        "particular point of law that is added to the text of a court"
+        "decision to aid readers in locating discussion of a legal"
+        "issue in an opinion. As the term implies, headnotes appear"
+        "at the beginning of the published opinion. Frequently, "
+        "headnotes are value-added components appended to "
+        "decisions by the publisher who compiles the "
+        "decisions of a court for resale. As handed down by "
+        "the court, a decision or written opinion does not contain "
+        "headnotes. These are added later by an editor not "
+        "connected to the court, but who instead works for a "
+        "legal publishing house. ",
+        blank=True,
+    )
+    summary = models.TextField(
+        help_text="A summary of what happened in the case. "
+        "Appears at the beginning of the case just "
+        "after the title of the case and court "
+        "information. Not identified as such.",
+        blank=True,
+    )
+    disposition = models.TextField(
+        help_text="Description of the procedural outcome of the case, "
+        "e.g. Reversed, dismissed etc. "
+        "Generally a short paragraph that appears "
+        "just after the summary or synopsis",
+        blank=True,
+    )
+    history = models.TextField(
+        help_text="History of the case (similar to the summary, "
+        "but focused on past events related to this case). "
+        "Appears at the beginning of the case just after "
+        "the title of the case and court information",
+        blank=True,
+    )
+    other_date = models.TextField(
```

Comment:
Should this be `other_dates`? I thought it could contain more than one?

Reply:
changed